lease-kubernetes/src/test/scala/akka/coordination/lease/kubernetes/KubernetesApiSpec.scala

```diff
@@ -53,7 +56,7 @@ class KubernetesApiSpec
 
   implicit val patience: PatienceConfig = PatienceConfig(testKitSettings.DefaultTimeout.duration)
 
-  val underTest = new KubernetesApiImpl(system, settings, "lease", "token", None)
+  val underTest = new KubernetesApiImpl(system, settings, "lease", () => Future.successful("token"), None)
```

Comment:
would it be possible to add a test with short token reload interval and verify that the `() => Future(token)` is called?